examples/testify/src/integration-test/groovy/gorm/tools/async/ParallelToolsSpecSpec.groovy

```diff
@@ -17,28 +17,19 @@ class ParallelToolsSpecSpec extends Specification {
         List<Map> list = createList(50)
 
         expect:
-        // Project.withSession {
-        //     Project.count() == 50
-        // }
         //starting org count
         Org.count() == 100
-
         list.size() == 50
 
         when:
         // FIXME #339 how is this working, transaction thats set on the test should not be rolling into parallelTools?
```

Comment:
lets deal with FIXME here while in here add comment on this question

Reply:
I havent been able to figureout it yet, so i did not remove fixme @basejump